src/store/authors/reducer.js

```diff
@@ -0,0 +1,32 @@
+import { mockedAuthorsList } from '../../constants';
+import { ADD_AUTHOR, DELETE_AUTHOR, SAVE_AUTHORS } from './types';
+
+export const authorsInitialState = [...mockedAuthorsList];
+
+// Use the initialState as a default value
+export const authorsReducer = (state = authorsInitialState, action) => {
+	// The reducer normally looks at the action type field to decide what happens
+	switch (action.type) {
+		// Do something here based on the different types of actions
+		case SAVE_AUTHORS:
+			return action.payload;
+
+		case ADD_AUTHOR:
+			return [...state, action.payload];
+
+		case DELETE_AUTHOR:
```

Comment:
avoid this eslint disable by defining your cases like:

case SAVE_AUTHORS: {

// your code

}